index.html

```diff
@@ -6,23 +6,53 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Albert's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+			rel="stylesheet"
+		/>
 </head>
 <body>
   <header class="header-container">
     <a href="#" class="logo">
         My Logo
     </a>
     <nav class="logo-button">
-        <a href="#"><img src="images/logo-button.png" alt="menu" width="18px" height="12px"></a>
+        <a href="#" class="humburger"><img src="images/logo-button.png" alt="menu" width="18px" height="12px"></a>
+        
+        <ul class="nav-bar">
+        <li>
+        <a href="#head" class="nav-link">Portfolio</a>
+        </li>
+        <li>
+        <a href="#about" class="nav-link">About</a>
+        </li>
+        <li>
+        <a href="#contact" class="nav-link">Contact</a>
+        </li>
+        <li>
+        </li>
+        <div class="close-menu"><img src="images/close.svg" alt="close" width="32px" height="32px"></div>
```

Comment:
- [ ] kindly consider giving this icon the same color as the one it have on figma to make your design an exact duplicate of the given one